src/surfaces/filtersRail/filtersRailRow.jsx

```diff
@@ -0,0 +1,201 @@
+import {
+    map,
+} from 'lodash';
+import ClassNames from 'classnames';
+import React, { useState } from 'react';
+import {
+    PROP_TYPES_ROW,
+} from './constants';
+import {
+    BEM_FILTERS_RAIL_ROW,
+    BEM_FILTERS_RAIL_ROW_COMONENT,
```

Comment:
Oh shoot.  I just realized "component" is misspelled in this constant.  It should be `BEM_FILTERS_RAIL_ROW_COMPONENT`.  Sorry I missed this earlier.

Reply:
All fixed up. Thanks!

Reply:
Looks good as of 7aa99cd.  Resolving.